### PR TITLE
[MM-9622] Added a custom link tooltip

### PR DESCRIPTION
**Pull request by kosgrz**
This PR adds a custom tooltip based on the code changes from this [PR](https://github.com/mattermost/mattermost-webapp/pull/2386).

tooltip

EDIT: updated image

### CODE REVIEW

**Comment by kosgrz** [comment]
Hi @jasonblais, I updated the README.md in the PR.

File: webapp/README.md [hunk 1]
```diff
@@ -115,3 +115,9 @@ This plugin registers a post dropdown action that, when clicked, triggers the ro
 This plugin registers a file upload action that, when clicked, triggers the root component:
 
 ![file upload](docs/file_upload.png)
+
+## Link Tooltip
+
+This plugin registers a link tooltip component, whose content is completely customizable and that is displayed when the mouse pointer is moved onto a link in a post.
```
Comment (jasonblais) [inline webapp/README.md]: By 'content', I assume this is the icon, colour and the text? If so, propose a minor update with

```
This plugin registers a link tooltip component, whose icon, text and colour are customizable. It is displayed when the mouse cursor hovers over a link in a post.
```

Comment (jasonblais) [reply webapp/README.md]: Ah okay. Then just minor proposed adjustment below and looks good to me :)

```
This plugin registers a link tooltip component, whose content is fully customizable. It is displayed when the mouse cursor hovers over a link in a post.
```

Comment (kosgrz) [reply webapp/README.md]: Done ;)

**Comment by cpanato** [comment]
will wait for https://github.com/mattermost/mattermost-webapp/pull/2386 to get merged

**Comment by hanzei** [comment]
Server PR is merged

**Comment by cpanato** [comment]
@kosgrz can you please update this branch with latest master?

**Comment by kosgrz** [comment]
@cpanato done.

**Comment by jasonblais** [comment]
@cpanato this one is ready to merge

**Pull request merged**